app/subnets/index.php

```diff
@@ -30,17 +30,6 @@
 # fetch recursive nameserver details
 $nameservers = (array) $Tools->fetch_object("nameservers", "id", $subnet['nameserverId']);
 
-# fetch all addresses and calculate usage
-if($slaves) {
-	$addresses = $Addresses->fetch_subnet_addresses_recursive ($subnet['id'], false);
-	$slave_subnets = (array) $Subnets->fetch_subnet_slaves ($subnet['id']);
-} else {
-	$addresses = $Addresses->fetch_subnet_addresses ($subnet['id']);
-}
-
-// get usage
-$subnet_usage  = $Subnets->calculate_subnet_usage ($subnet, true);
```

Comment:
Hi, this is needed to properly calculate subnet usage.
<img width="305" alt="screen shot 2017-05-08 at 07 56 48" src="https://cloud.githubusercontent.com/assets/12947690/25791953/f31546a8-33c3-11e7-88e7-946d1be278fd.png">